jaxrs-spec/src/main/asciidoc/chapters/introduction/_status.adoc

```diff
@@ -11,20 +11,20 @@
 [[status]]
 === Status
 
-This is the final release of version 2.1. The issue tracking system for
+This is the final release of version 3.0. The issue tracking system for
 this release can be found at:
 
-https://github.com/jax-rs/api/issues
+https://github.com/eclipse-ee4j/jaxrs-api/issues
 
 The corresponding Javadocs can be found online at:
 
-https://jax-rs.github.io/apidocs/2.1
+https://jakarta.ee/specifications/restful-ws/3.0/apidocs/
 
 The reference implementation can be obtained from:
 
-https://jersey.github.io
+https://eclipse-ee4j.github.io/jersey/
```

Comment:
Thanks for updating the link.  My comment here is not necessarily about your change, but about whether we should list "reference implementation" as such.  My understanding is that Jakarta is moving away from _reference_ implementation, and instead is moving toward _compatible_ implementations - similar to MicroProfile.  Should we turn this into a list of compatible implementations?  Or maybe just remove it since we don't know yet who will be implementing it?  Wdyt?

Reply:
Good point. I also wondered if we should change the wording. However, I noticed that there are several wordings in the spec that need to be adjusted and therefore decided to do this in a subsequent pull request and/or discuss this issue on the mailing list first. For example:

- "During the course of this **JSR** we received..."
- "**Expert Group** Members"
- "The **reference implementation** can be obtained"

Reply:
I agree, we should clean this up. @chkal If you can create an issue with all these I can help.

Reply:
Ok, great! I'll file a separate issue for that.